doc/manual/src/plugins.md

```diff
@@ -0,0 +1,186 @@
+# Plugins
+
+This chapter describes all plugins present in hydra
+
+### Inputs
+
+Hydra supports the following inputs
+
+- Bazaar input
+- Darcs input
+- Git input
+- Mercurial input
+- Path input
+
+## Bitbucket pull requests
+
+Create jobs based on open bitbucket pull requests
+
+configuration options:
+- `bitbucket_authorization.<owner>`
+
+## Bitbucket status
+
+Sets Bitbucket CI status
+
+configuration options:
+- `enable_bitbucket_status`
+- `bitbucket.username`
+- `bitbucket.password`
+
+## Circle CI Notifcation
+
+Sets Circle CI status
+
+configuration options:
+- `circleci.[].jobs`
```

Comment:
I am open for suggestions regarding the notation. Some options can be repeated, some take a key, others are scalar.